### PR TITLE
Feature/jsonapi rails support

### PR DESCRIPTION
**Pull request by nevenRakonic**
Support the rendering method required by jsonapi-rails gem

### CODE REVIEW

**Comment by DamirSvrtan** [comment]
@nikone can you please merge this pull request if it's okay with you and release a new gem version?

File: spec/lib/json_api_responders/config/config_spec.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+describe JsonApiResponders::Config do
+  let(:config) { JsonApiResponders.config }
+
+  context 'when config render method is not set' do
+    it 'render_method is set to default render method' do
+      expect(config.render_method).to eq JsonApiResponders::Config::DEFAULT_RENDER_METHOD
+    end
+    after { clear_config }
```
Comment (nikone) [inline spec/lib/json_api_responders/config/config_spec.rb]: All rspec hooks should be defined before the tests.

File: spec/lib/json_api_responders/config/config_spec.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    before do
+      JsonApiResponders.configure do |config|
+        config.render_method = :jsonapi
+      end
+    end
+
+    it 'render_method is set to :jsonapi' do
+      expect(config.render_method).to eq :jsonapi
+    end
+    after { clear_config }
```
Comment (nikone) [inline spec/lib/json_api_responders/config/config_spec.rb]: same as above

File: spec/lib/json_api_responders/config/config_spec.rb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    subject do
+      JsonApiResponders.configure do |config|
+        config.render_method = :invalid_render_method
+      end
+    end
+
+    it 'raises an error' do
+      expect{ subject }.to raise_error(JsonApiResponders::Errors::InvalidRenderMethodError)
+    end
+    after { clear_config }
```
Comment (nikone) [inline spec/lib/json_api_responders/config/config_spec.rb]: same as above

**Pull request merged**